### ISSUE TITLE
Country flags and icons are broken in 7.6.0

### ISSUE THREAD

**Issue by thomaszbz** [mblaschke/TYPO3-metaseo#257]
Flags are rendered as a red X in Metatag/Seo section

Metatag/SEO section:
![auswahl_008](https://cloud.githubusercontent.com/assets/11200727/16707075/00eba7fc-45c4-11e6-8234-0d1ae53b85a4.png)

Sitemap section:
![auswahl_007](https://cloud.githubusercontent.com/assets/11200727/16707074/00e8f11a-45c4-11e6-8fd7-01e62b4a894d.png)

Languages are configured and TYPO3 shows the country flags:
![auswahl_009](https://cloud.githubusercontent.com/assets/11200727/16707076/00ec0b0c-45c4-11e6-8ab0-c3a422443876.png)

This seems to be specific to TYPO3 7.6. Tested with metaseo 2.0.0, TYPO3 7.6.10-dev ec361d099fe612ab2b8ed1d0c3c80f0a01eb0fb2 (2016-06-20).

We use css classes as follows:

```
<span class="t3-icon t3-icon-flags t3-icon-flags-de t3-icon-de"></span>&nbsp;Deutsch
```

If we use the old `IconUtility` in 7.6 instead of css classes, country flags and icons are still broken. As of 7.6.10 only `IconFactory` and `IconRegistry` work right which basically means that there is no usable deprecation concept available to support both, 6.2 and 7.6.

**Comment by thomaszbz** [comment]
This has been fixed recently in TYPO3 core:

https://forge.typo3.org/issues/68217
http://review.typo3.org/41478

**Comment by thomaszbz** [comment]
In a later patch the css classes for the flags we use got removed. This happened in an unreleased 7.6.10-dev via https://github.com/TYPO3/TYPO3.CMS/commit/58c46d75099acd8449aded959efe8f031cec13d0

https://github.com/TYPO3/TYPO3.CMS/commits/TYPO3_7-6/typo3/sysext/core/ext_tables.php

**Comment by thomaszbz** [comment]
At least the flags are broken in metaseo 2.0.0 together with TYPO3 7.6 because of a change in TYPO3 7.6. This is not a regression in metaseo 2.0.0 but maybe we need a compatibility fix for 7.6.x or TYPO3 7.6 core needs to be fixed.

I filed a forge-issue to see if the (breaking) change was intended behaviour and if we need to patch this _now_ or if TYPO3 patches it so that we can patch it as soon as we drop support for 6.2 in favour of 8.x:

https://forge.typo3.org/issues/76996

**Comment by thomaszbz** [comment]
Thanks to Frank Naegler, the [issue](https://forge.typo3.org/issues/76996) got fixed in [TYPO3 CMS 7.6.11](TYPOhttps://wiki.typo3.org/TYPO3_CMS_7.6.11).

However, we still need to adopt portions of our javascript code at least for two reasons:

- CSS classes still are not available any more
- DOM has changed which renders our buttons unclickable
